### ISSUE TITLE
disabled TouchableHighlight still receives keyboard focus

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-windows#8875]
### Problem Description

from: https://github.com/microsoft/react-native-windows/issues/8781

Disabled TouchableHighlight still receives keyboard focus if onPress is defined.

### Steps To Reproduce

1. Set `disabled={true}` and define an onPress handler for a TouchableHighlight component
2. tab through and observe how it still receives keyboard focus
3. vs if you remove the onPress it behaves as expected and no longer gets keyboard focus

### CLI version

6.1.0

### Environment

```markdown
npx react-native info
```

### Snack, code example, screenshot, or link to a repository

```js    
      <View>
            <TouchableHighlight disabled={true} onPress={this._handlePress}>
              <Text>text within a disabled touchable</Text>
            </TouchableHighlight>
            <TouchableHighlight onPress={this._handlePress}>
              <Text>text within a disabled touchable</Text>
            </TouchableHighlight>
      </View>
```
The disabled TouchableHighlight still receives keyboard focus

**Comment by AgneLukoseviciute** [comment]
explicitly setting focusable to false can be used as a workaround. I think we'll want to do something similar to https://github.com/microsoft/react-native-windows/pull/9840 so that when the disabled prop is set to true, we make the control not focusable/inaccessible. This will mimic the behavior in XAML
